python/legate_dataframe/lib/core/column.pyi

```diff
@@ -3,34 +3,21 @@
 
 from typing import Any
 
-import cudf
 import legate.core
 import pyarrow as pa
-from cudf._typing import DtypeObj
 from legate.core import StoreTarget
-from pylibcudf.column import Column as PylibcudfColumn
-from pylibcudf.scalar import Scalar as PylibcudfScalar
-
-from legate_dataframe.lib.core.data_type import DataType
-
-_CudfTypes = (
-    cudf.core.column.column.ColumnBase
-    | cudf.core.series.Series
-    | PylibcudfColumn
-    | PylibcudfScalar
-)
```

Comment:
Hmmm, too bad, but I am not aware of a solution (doesn't seem worthwhile to spell out a string annotation.)